ballerina/websocket_service.bal

```diff
@@ -24,17 +24,17 @@ isolated service class WsService {
     private final Engine engine;
     private final readonly & __Schema schema;
     private final Context context;
-    private string[] activeConnections;
+    private map<string> activeConnections;
     private final readonly & map<string> customHeaders;
     private boolean initiatedConnection;
 
-    isolated function init(Engine engine, __Schema & readonly schema,  map<string> & readonly customHeaders,
-                           Context context) {
+    isolated function init(Engine engine, __Schema & readonly schema, map<string> & readonly customHeaders,
+            Context context) {
```

Comment:
Incorrect indentation?

Reply:
No, this is formatted with ballerina

Reply:
This is not what we used in other places. we have used the previous indentation in the function definition.

Reply:
Ballerina formatter does weird formatting when a single line breaks into two. Let's stick with the existing indentations for now.

Reply:
reverted